### DISCUSSION TITLE
[@types/jsonwebtoken] dependency on ms

### DISCUSSION THREAD

**Discussion by aqeelat** [DefinitelyTyped/DefinitelyTyped#71394] [Issues with a @types package] [answered]
jsonwebtoken uses [@zeit/ms](https://github.com/zeit/ms.js) in:
https://github.com/DefinitelyTyped/DefinitelyTyped/blob/ff46ba2eac810fe812beac0d8c0894bfad853a52/types/jsonwebtoken/index.d.ts#L42-L45
That library exports its types: https://github.com/vercel/ms/blob/8b5923d1d86c84a9f6aba8022d416dcf2361aa8d/src/index.ts#L44-L47
Why not use it here instead of current implementation?

I could open a PR for it. Should I add it as a direct dependency?

p.s. @types/ms won't work
https://github.com/DefinitelyTyped/DefinitelyTyped/blob/ff46ba2eac810fe812beac0d8c0894bfad853a52/types/ms/index.d.ts#L1-L18

[a comment by an automated account is not shown]

**Reply by nandi95** [reply, marked as answer]
I don't see, why not.

**Reply by aqeelat** [reply]
my concern is using the same ms version as is installed by the user to not cause duplicate dependencies. how should we approach that?
